src/gpo/assets/PowerToys.admx

```diff
@@ -269,7 +269,17 @@
         <decimal value="0" />
       </disabledValue>
     </policy>
-    <policy name="ConfigureEnabledUtilityScreenRuler" class="Both" displayName="$(string.ConfigureEnabledUtilityScreenRuler)" explainText="$(string.ConfigureEnabledUtilityDescription)" key="Software\Policies\PowerToys" valueName="ConfigureEnabledUtilityScreenRuler">
+	  <policy name="ConfigureEnabledUtilityRegistryPreview" class="Both" displayName="$(string.ConfigureEnabledUtilityRegistryPreview)" explainText="$(string.ConfigureEnabledUtilityDescription)" key="Software\Policies\PowerToys" valueName="ConfigureEnabledUtilityRegistryPreview">
+		  <parentCategory ref="RegistryPreview" />
+		  <supportedOn ref="SUPPORTED_POWERTOYS_0_64_0" />
```

Comment:
This needs to be changed before merging. I think we will need a new entry for "SUPPORTED_POWERTOYS_0_69_0".

Reply:
Do we need to update the "Last updated" comment a few lines later too?